wx/lib/asyncio.py

```diff
@@ -0,0 +1,336 @@
+#----------------------------------------------------------------------
+# Name:        wx.lib.asyncio
+# Purpose:     Coroutine support for event handlers
+#
+# Author:      XU Guang-zhao 徐广钊
+#
+# Created:     8-Dec-2018
+# Copyright:   (C) 2018 XU Guang-zhao 徐广钊
+# Licence:     wxWidgets license
+#
+# Tags:        documented
+#
+#----------------------------------------------------------------------
+
+"""
+Add ``wx.BindAsync()`` and other supporting classes to enable coroutine_
+support for wxPython.
+
+.. _coroutine: https://docs.python.org/3/library/asyncio-task.html#coroutine
+
+Description
+===========
+Before the introduction of coroutines, developers for GUI applications have to ways
+to handle time-consuming tasks. One of them is ``wx.Yield()`` which periodically
```

Comment:
It's periodic only if you call wx.Yield periodically. You can just remove "periodically" here